### ISSUE TITLE
Nuxt v2.10 - TypeError: Cannot read property 'keyName' of undefined`

### ISSUE THREAD

**Issue by perezd** [nuxt/nuxt.js#6506]
### Version

[v2.10.0](https://github.com/nuxt.js/releases/tag/v2.10.0)

### Reproduction link

[https://github.com/nuxt/create-nuxt-app/issues/355#](https://github.com/nuxt/create-nuxt-app/issues/355#)

### Steps to reproduce

Repro steps present at link.

### What is expected ?

Working server/default project.

### What is actually happening?

This shows up in DevTools:
https://camo.githubusercontent.com/8f11ac3655907c26a60f8c161c006fe67f5f3424/68747470733a2f2f6d656469612e646973636f72646170702e6e65742f6174746163686d656e74732f3437333430363030343735313439393237352f3632383938363830373936363439383832372f756e6b6e6f776e2e706e67

### Additional comments?

Issue identified using create-nuxt-app:
https://github.com/nuxt/create-nuxt-app/issues/355#

<div align="right"><sub><em>This bug report is available on <a href="https://cmty.app/nuxt">Nuxt</a> community (<a href="https://cmty.app/nuxt/nuxt.js/issues/c9854">#c9854</a>)</em></sub></div>

**Comment by Tantawi** [comment]
Same issue here with 2.10.0 release.

**Comment by DanJRWalsh** [comment]
Got the same thing after upgrading to 2.10.0 on a project that works fine on 2.9.2.

Had to revert back in the meantime.

**Comment by zwik** [comment]
Having a similar issue with a pristine SSR project:
![image](https://user-images.githubusercontent.com/1595807/66108524-10844c80-e5c3-11e9-931c-b4bddc9bc9f7.png)

**Comment by catisnorm** [comment]
With no chosen UI framework works well.

**Comment by terean-dspd** [comment]
I'm having the same poblem but only on Ubuntu. On Windods 10 it works well by the way.

**Comment by perezd** [comment]
Can confirm I'm on Ubuntu

**Comment by zwik** [comment]
> 
> 
> I'm having the same poblem but only on Ubuntu. On Windods 10 it works well by the way.

At first I used create-nuxt-app as following:
```
package manager: Yarn
UI framework: Vuetify
server framework: Express
Nuxt modules: Axios + PWA
Lint: ESLint + Prettier
test framework: Jest
rendering mode: SSR
```

I'm having the problem on a Windows 10 machine, can't confirm for Linux though. I had the project configured to use Vuetify and the above problem occurred. I checked if the same happens when no UI framework is being selected and indeed, it then works.

```
I then used this config:
package manager: Yarn
UI framework: none
server framework: Express
Nuxt modules: Axios + PWA
Lint: ESLint + Prettier
test framework: Jest
rendering mode: SSR
```

**Comment by pimlie** [comment]
You can mitigate this issue by installing/upgrading to vue-meta `v2.3.0` manually

Also see: https://github.com/vuetifyjs/vuetify/issues/9181

**Comment by zwik** [comment]
For me the issue is now resolved. I'm not sure if installing `vue-meta` is helping. When I do a pristine SSR project it works. An inspection in the `yarn.lock` learns me that `vue-meta` 2.3.0 is installed so I guess that indeed helped.

**Comment by catisnorm** [comment]
Problem disappeared for SPA mode for me in [Nuxt 2.10.1](https://github.com/nuxt/nuxt.js/releases/tag/v2.10.1), `vue-meta` is updated in this release

**Comment by pimlie** [comment]
Thanks, the issue is solved in `v2.10.1` indeed. Will close this issue because of that
